operator/gitops/argocd/pipeline-service/pipelines-as-code/kustomization.yaml

```diff
@@ -20,3 +20,30 @@ patchesJson6902:
         path: "/metadata/labels"
         value:
           argocd.argoproj.io/managed-by: openshift-gitops
+  - target:
+      kind: Deployment
+      name: pipelines-as-code-controller
+      namespace: pipelines-as-code
+    patch: |-
+      - op: replace
+        path: "/spec/template/spec/containers/0/image"
+        value:
+          registry.redhat.io/openshift-pipelines/pipelines-pipelines-as-code-rhel8@sha256:6ba7db7a829bdac4db7554a4643283d0e97fc7a407ed7b6297888124fb5af3f3
```

Comment:
I'll add what I mentioned in planning @Roming22 for completeness.

Per https://catalog.redhat.com/software/containers/openshift-pipelines/pipelines-pipelines-as-code-rhel8/61f93fe495bac8d0de150ad8 how about trying `v1.10.0-6` instead of the specific sha

The way those tags work in registry.redhat.io,  `v1.10.0-6` is not a rolling tag, but will only ever point to the sha it is initially seeded with.

The makes it clear to humans when looking at the deployment which version we are using here.

Reply:
Fixed.